pydis_site/apps/content/resources/guides/pydis-guides/contributing.md

```diff
@@ -91,26 +91,57 @@ Our projects on Python Discord are open source and [available on Github](https:/
 
-#### Guidelines for the @Contributors Role
+  Check out these [**resources to get started using Git**](./working-with-git/).
 
-One question we get a lot is what the requirements for the **@Contributors** role are.
-As it’s difficult to precisely quantify contributions, we’ve come up with the following guidelines for the role:
+### 2. Fork the repo
+  GitHub is a website based on the Git version control system that stores project files in the cloud. The people working on the project can use GitHub as a central place for sending their changes, getting their teammates' changes, and communicating with each other. Forking the repository that you want to work on will create a copy under your own GitHub account. You'll make your changes to this copy, then later we can bring them back to the PyDis repository.
 
-- The member has made several significant contributions to our projects.
-- The member has a positive influence in our contributors subcommunity.
+  Check out our [**guide on forking a GitHub repo**](./forking-repository/).
+### 3. Clone the repo
+  Now that you have your own fork you need to be able to make changes to the code. You can clone the repo to your local machine, commit changes to it there, then push those changes to GitHub.
 
-The role will be assigned at the discretion of the Admin Team in consultation with the Core Developers Team.
+  Check out our [**guide on cloning a GitHub repo**](./cloning-repository/).
+
+### 4. Set up the project
+  You have the source code on your local computer, now how do you actually run it? We have detailed guides on setting up the environment for each of our projects:
+
+  * [**Sir Lancebot**](./sir-lancebot/)
+
+  * [**Python Bot**](./bot/)
+
+  * [**Site**](./site/)
+
+### 5. Read our Contributing Guidelines
+  We have a few short rules that all contributors must follow. Make sure you read and follow them while working on our projects.
+
+  [**Contributing Guidelines**](./contributing-guidelines/).
+
+  As mentioned in the Contributing Guidelines, we have a simple style guide for our projects based on PEP 8. Give it a read to keep your code consistent with the rest of the codebase.
+
+  [**Style Guide**](./style-guide/)
+
+### 6. Create an issue
+  The first step to any new contribution is an issue describing a problem with the current codebase or proposing a new feature. All the open issues are viewable on the GitHub repositories, for instance here is the [issues page for Sir Lancebot](https://github.com/python-discord/sir-lancebot/issues). If you have an idea that you want to implement, open a new issue (and check out our [**guide on writing an issue**](./issues/)). Otherwise you can browse the unassigned issues and ask to be assigned to one that you're interested in, either in the comments on the issue or in the`#dev-contrib`channel on Discord.
```

Comment:
https://www.pythondiscord.com/pages/guides/pydis-guides/contributing/bot/#issues has a link to the channel, could use it here as well.